plugins/rts54hub/rts54hub.quirk

```diff
@@ -0,0 +1,23 @@
+# RTS54 HS
+[DeviceInstanceId=USB\VID_0BDA&PID_5423]
+Plugin = rts54hub
+FirmwareSizeMin = 0x10000
+FirmwareSizeMax = 0x10000
+
+# RTS54 SS
+[DeviceInstanceId=USB\VID_0BDA&PID_0423]
+Plugin = rts54hub
+FirmwareSizeMin = 0x10000
+FirmwareSizeMax = 0x10000
+
+# Custom
+[DeviceInstanceId=USB\VID_0BDA&PID_5420]
+Plugin = rts54hub
+FirmwareSizeMin = 0x10000
+FirmwareSizeMax = 0x10000
+
+# Custom
+[DeviceInstanceId=USB\VID_0BDA&PID_5423]
```

Comment:
@AnyProblem -- this is the ID of the board you sent me -- what should I call it? "RTS5423 Development Board" perhaps?

Reply:
Yes, RTS5423 Development Board is ok